channels/qa.m3u

```diff
@@ -1,6 +1,8 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidearab.xml.gz"
 #EXTINF:-1 tvg-id="Al Jazeera Arabic ARB" tvg-name="Al Jazeera Arabic ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/x9TQYI3.jpg" group-title="News",Al Jazeera Arabic
 https://live-hls-web-aja.getaj.net/AJA/index.m3u8
+#EXTINF:-1 tvg-id="Al Jazeera Documentary ARB" tvg-name="Al Jazeera Documentary ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/UhEKMdb.png" group-title="Documentary",Al Jazeera Documentary
+https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
```

Comment:
That's what I personally see. 
<img width="738" alt="Captura de pantalla 2020-03-25 a las 20 59 34" src="https://user-images.githubusercontent.com/7253922/77569820-da71b680-6edb-11ea-9861-ad4fe1519365.png">
And after 40 seconds the broadcast goes off completely.

Reply:
It's blocked in your country

> #EXTM3U
#EXT-X-VERSION:6
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-STREAM-INF:BANDWIDTH=6324320,RESOLUTION=1920x1080,CODECS="avc1.640028,mp4a.40.2"
01.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=3120800,RESOLUTION=1280x720,CODECS="avc1.64001f,mp4a.40.2"
02.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1831872,RESOLUTION=960x540,CODECS="avc1.64001f,mp4a.40.2"
03.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1013696,RESOLUTION=746x420,CODECS="avc1.64001e,mp4a.40.2"
04.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=813664,RESOLUTION=640x360,CODECS="avc1.64001e,mp4a.40.2"
05.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=428640,RESOLUTION=426x240,CODECS="avc1.640015,mp4a.40.2"
06.m3u8
![image](https://user-images.githubusercontent.com/59766876/77585498-7f8c8f00-6edc-11ea-8ced-3df64b398023.png)